File: code/9.py
for a in range (1, 1001):
	for b in range (1, 1001):
		for c in range (1, 1001):
			ispyt = False
			if a*a+b*b == c*c:
				ispyt = True
			if a+b+c == 1000 and ispyt:
				print(a,b,c,"\n")
				prod = a*b*c
			c += 1
		b += 1
	a += 1

print(prod)

File: code/9.cc
#include <iostream>
#include <cmath>
#include <algorithm>
using namespace std;

int main(){
	int a, b, c, prod;
	a = 1;
	while(a <= 1000){
		b = 1;
		while(b <= 1000){
			c = 1;
			while(c <= 1000){
				bool ispyt = false;
				if((a*a+b*b) == (c*c))
					ispyt = true;
				if((a+b+c) == 1000 &&ispyt){
					cout<<a<<" "<<b<<" "<<c<<endl;
					prod = a*b*c;
				}
				c++;
			}
			b++;
		}
		a++;
	}
	cout<<prod<<endl;
	return 0;
}
